micro-infra-spring-base/src/main/groovy/com/ofg/infrastructure/web/logging/EnableRequestResponseLogging.java

```diff
@@ -19,4 +19,4 @@
 @Target(ElementType.TYPE)
 @Retention(RetentionPolicy.RUNTIME)
 @Import(RequestLoggingConfiguration.class)
-public @interface EnableRequestBodyLogging {}
+public @interface EnableRequestResponseLogging {}
```

Comment:
maybe `EnableObfuscatedLogging`? because there is logging without this annotation...